rest/clients/cxf/owasp-suppressions.xml

```diff
@@ -28,4 +28,12 @@
         <cpe>cpe:/a:springsource:spring_framework</cpe>
         <cpe>cpe:/a:pivotal_software:spring_framework</cpe>
     </suppress>
+    <suppress>
+        <!-- Only present when JDOM 1.x or 2.x jar is in the classpath. JDOM has been excluded. -->
```

Comment:
:question: What happens if a client does have that jar on their classpath? How can you be sure they won't?

Reply:
If someone pulled in this dependency, OWASP dependency check would fail on that project. This suppression should not be inherited by other projects. See https://github.com/connexta/multi-int-store/pull/66/commits/b96408eee7c1d8fd03493f7021f2522651fa4d94.

Reply:
My concern is that we are then hiding the fact that there is a vulnerability that they could introduced in their code elsewhere just because we are suppressing it here generically. As I already discussed with @lessarderic, doing so without being very specific about what part of the code we are suppressing it for can be dangerous.